params/extras/config.go

```diff
@@ -0,0 +1,352 @@
+// (c) 2024 Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package extras
+
+import (
+	"encoding/json"
+	"fmt"
+	"math/big"
+
+	"github.com/ava-labs/avalanchego/snow"
+	"github.com/ava-labs/avalanchego/upgrade"
+	"github.com/ava-labs/avalanchego/utils/constants"
+	"github.com/ava-labs/libevm/common"
+	ethparams "github.com/ava-labs/libevm/params"
+	"github.com/ava-labs/subnet-evm/commontype"
+	"github.com/ava-labs/subnet-evm/utils"
+)
+
+var (
+	DefaultFeeConfig = commontype.FeeConfig{
```

Comment:
I moved this so it can be used in tests, but also put it back in it's original location to avoid having to change all params.DefaultFeeConfig's in the code.

Reply:
Maybe that's out of scope, but can't we unexport and put these configurations `DefaultFeeConfig` and `TestChainConfig` in a `_test.go` file, since these are only used in tests in this package?

Reply:
It is re-exported (and then referred to in many places) as `params.DefaultFeeConfig`